dotnet/samples/04.ai.a.teamsChefBot/files/github.txt

```diff
@@ -0,0 +1,19 @@
+# Teams AI Library
+
+Welcome to the Teams AI Library .NET package! 
```

Comment:
Are we planning to add these txt files from `getting-started` into each sample?

Reply:
Nope the Teams Chef bot will use these files as context through the Kernel Memory datasource.